src/commands/basemaps-mapsheet/create-mapsheet.ts

```diff
@@ -22,9 +23,9 @@ export function isGzip(b: Buffer): boolean {
  *
  * If the file ends with .gz or is a GZIP like {@link isGzip} file it will automatically be decompressed.
  */
-async function readConfig(config: string): Promise<ConfigBundled> {
-  const obj = await fsa.read(config);
-  if (config.endsWith('.gz') || isGzip(obj)) {
+async function readConfig(config: URL): Promise<ConfigBundled> {
+  const obj = await fsa.read(config.href);
```

Comment:
its not safe to use `.href` here. If its a file url it has to be converted with `path.fileURLToPath`.

does `fsa.read` not accept a URL? would imply `fsa` is out of date?

Reply:
> does `fsa.read` not accept a URL? would imply `fsa` is out of date?

[Yep](https://github.com/linz/argo-tasks/pull/817); that PR needs a bunch of fixes, though.